Assets/Scripts/Tile.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Tile
+{
+    private bool pathTile = false;
+    private bool taken = false;
+    private Vector3 position = new Vector3(0.0f, 0.0f, 0.0f);
```

Comment:
Any reason you have position as a variable here instead of just getting it from transform.position? From what I can tell they'll have the same value.